pkg/kubelet/kubelet.go

```diff
@@ -2673,6 +2673,12 @@ func getPodReadyCondition(spec *api.PodSpec, containerStatuses []api.ContainerSt
 			unknownContainers = append(unknownContainers, container.Name)
 		}
 	}
+
+	// In case of unexist unknowContainers, If pod has derminated successed and it has unreadyContainers, just return PodCompleted
+	if podPhase == api.PodSucceeded && len(unknownContainers) == 0 && len(unreadyContainers) != 0 {
```

Comment:
1:The funcition getPodReadyCondition should return true when the pod has succeeded and there are no unready containers; is my view, what do you think?
2:If this come into existence, i think we should add the len(unreadyContainers) != 0, if no , we will directly return false when pod has success, but no unready container.
sorry if i am understand wrong, i will change the code after your apply. 
thanks very much.
@erictune